proto/viam/app/build/v1/build.proto

```diff
@@ -23,3 +27,51 @@ message StartBuildResponse {
   // github's ID for the build
   optional string github_id = 1;
 }
+
+message GetLogsRequest {
+  // module_id as prefix:name pair
+  string module_id = 1;
+  string job_id = 2;
+}
+
+// GetLogsReponse is a streaming endpoint that may have multiple messages that belong
+// to the same build_step if there are too many log lines to fit into a single gRPC
+// response
+message GetLogsResponse {
+  string build_step = 1;
+  repeated string log_lines = 2;
+}
+
+enum JobStatus {
+  JOB_STATUS_UNSPECIFIED = 0;
+  // STARTED = in the database, being queued / prepared for execution
+  JOB_STATUS_STARTED = 1;
```

Comment:
vendor calls this state 'queued' to make it clear it hasn't started yet

up to you (docstring is clear enough) but wouldn't mind changing this to make clear the job is pending

Reply:
:+1: Will update.

Reply:
Updated